<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/app.png</normaloff>icons/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="menu_desplegable" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>751</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(230, 230, 230);
	color: black;
}

QPushButton{
	height: 30px;
	border:none;
}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/app.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CheckSalud</string>
     </property>
    </widget>
    <widget class="QPushButton" name="usuarios">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{
	background-color: rgb(255, 255, 255);
	border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/pacients.png</normaloff>
       <normalon>icons/on_pacientes.png</normalon>icons/pacients.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="informacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{
	background-color: rgb(255, 255, 255);
	border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_informacion.png</normaloff>
       <normalon>icons/on_informacion.png</normalon>icons/icon_informacion.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="salir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_salir.png</normaloff>
       <normalon>icons/on_salir.png</normalon>icons/icon_salir.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>791</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="btnBuscarPacienteRegistrado">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>50</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	background-color: rgb(43, 130, 130);
	color: white;
	border: 1p solid grayx;
	border-radius: 10px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Factores</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>16</width>
       <height>441</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="buscarResultadoPaciente">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Buscar resultados...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rut del paciente:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="resultadoPaciente">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>301</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExportarResultado">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>330</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	
	background-color: rgb(134, 0, 0);
	color: white;
	border: 1p solid grayx;
	border-radius: 10px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Exportar Documento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelRespirar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelAlcohol">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelPolvo">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelRiesgoGen">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelObesidad">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelCigarro">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Datos personales</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Género:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Edad:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbGeneroPredecir">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Seleccionar género</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Masculino</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femenino</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtEdadPredecir">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Consumo Alcohol</string> 
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alergia al polvo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Riesgo genetico</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Obesidad</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Consumo de cigarro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dificultad al respirar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dolor de Pecho</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNivelPecho">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPredecir">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>400</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	background-color: rgb(43, 130, 130);
	color: white;
	border: 1p solid grayx;
	border-radius: 10px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Predecir</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRutPacientePredecir">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>50</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Ingresar el rut...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rut del paciente:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>791</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(175, 175, 175);</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRutPaciente">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Ingresar el rut...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Apellidos:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNombrePaciente">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Ingresar el nombre...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rut del paciente:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtApellidosPaciente">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Ingresar apellidos...</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>100</y>
       <width>781</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Edad:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEdadPaciente">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Género:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbGeneroPaciente">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Seleccionar género</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Masculino</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femenino</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Teléfono:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNumeroPaciente">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Número de contacto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIngresarPaciente">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgb(0, 170, 0);
	color: white;
	border: 1p solid grayx;
	border-radius: 10px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salir</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
